src/main/js/field/ReadOnlyField.js

```diff
@@ -29,12 +29,14 @@ class ReadOnlyField extends Component {
 }
 
 ReadOnlyField.propTypes = {
+    id: PropTypes.string,
```

Comment:
Is this property used?

Reply:
because of this code it will `{...this.props}` set the id property on the LabeledField.  Have to confirm LabeledField is doing the right thing.